components/TicketsSection.js

```diff
@@ -29,18 +29,31 @@ function TicketsTable () {
   )
 }
 
-function TicketsRow ({ title, date }) {
+function TicketsRow ({ title, date, soldout }) {
   return (
     <div className='tickets-row'>
-      <div className='title'>{title}</div>
-      <div className='date'>{date}</div>
+      { soldout && <span className='indicator-soldout' /> }
+      <div className={`title ${soldout && 'text-soldout'}`}>{title}</div>
+      <div className={`date ${soldout && 'text-soldout'}`}>{date}</div>
```

Comment:
@our9planets This way will generate the `undefined` class too na krub. 

credit @soraxtend - https://github.com/reactbkk/2.0.0/pull/43#discussion_r117229732